core/src/modules/ops/profileProperty.ts

```diff
@@ -32,14 +32,14 @@ export namespace ProfilePropertyOps {
       case "url":
         return formatURL(value.toString());
       case "boolean":
-        if (![true, false, 0, 1, "true", "false"].includes(value)) {
-          throw new Error(`${value} is not a valid boolean value`);
```

Comment:
That `"True"` is really odd - wouldn't this line have thrown?  And down below in `getValue` it would have been coerced to `false`...

Either way... good change!

Reply:
yes, it would have thrown, so I stopped them and had them lowercase it with custom SQL. But then made this so it's not an issue.